### PR TITLE
Fix sqs dashboard and add s3 dashboard image to doc

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
By looking at AWS SQS monitoring console for a specific queue, most of the metrics they are showing applies statistic method `Sum`. 
Screen Shot 2019-05-01 at 12 37 02 PM
This is why I switched statistic method from average to sum in this PR. 

Thanks @sayden for testing the sqs dashboard and found a bug in the filter section. This PR also fixed that. 

Also s3 dashboard is added but not shown in the documentation. This PR adds the example dashboard into documentation as well.

### CODE REVIEW

**Comment by kaiyan-sheng** [comment]
Looking at various statistic method that AWS puts by default on their dashboards made me start thinking it might be useful to add a config option into aws module for different statistic methods when querying cloudwatch metrics.  I will start an issue discussing this.

**Comment by sayden** [comment]
Getting this error when trying to open the dashboard. However I don't see anything wrong in the JSON file 
image

File: x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-sqs-overview.json [hunk 1]
```diff
@@ -5,111 +5,10 @@
...
         "hits": 0,
         "kibanaSavedObjectMeta": {
-          "searchSourceJSON": {
-            "filter": [],
-            "query": {
-              "language": "kuery",
-              "query": ""
-            }
-          }
+          "searchSourceJSON": "{\"filter\":[],\"query\":{\"language\":\"kuery\",\"query\":\"\"}}"
```
Comment (sayden) [inline x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-sqs-overview.json]: This seems like the offending line :confused:

Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/aws/_meta/kibana/7/dashboard/Metricbeat-aws-sqs-overview.json]: yeah turned out I was exporting dashboard without decoding them.

**Comment by kaiyan-sheng** [comment]
jenkins, test this please

**Pull request merged**